src/icons/Celebration.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
+import colors from 'src/styles/colors'
+import Svg, { Path } from 'svgs'
+
+interface Props {
+  height?: number
+  width?: number
```

Comment:
Are we only conna use square images? if so we should probably use a single prop to avoid improper use.
```suggestion
  size?: number
```